src/components/Inventory/view/WithdrawalModal.tsx

```diff
@@ -69,10 +69,22 @@ export default function WithdrawalsModal(props: WithdrawalsModalProps): JSX.Elem
   const MANDATORY_FIELDS = ['purpose', 'withdrawnDate'] as const;
   type MandatoryField = typeof MANDATORY_FIELDS[number];
 
+  const validateBatchWithdrawal = () => {
+    if (record.batchWithdrawals && record.batchWithdrawals[0]) {
+      if (!record.batchWithdrawals[0].readyQuantityWithdrawn) {
```

Comment:
```
    BatchWithdrawalPayload: {
      batchId: number;
      germinatingQuantityWithdrawn?: number;
      notReadyQuantityWithdrawn: number;
      readyQuantityWithdrawn: number;
    };
```
is notReadyQuantityWithdrawn required too?

Reply:
i think it can be 0